go/backend/stock/file/readonly_test.go

```diff
@@ -0,0 +1,151 @@
+// Copyright (c) 2024 Fantom Foundation
+//
+// Use of this software is governed by the Business Source License included
+// in the LICENSE file and at fantom.foundation/bsl11.
+//
+// Change Date: 2028-4-16
+//
+// On the date above, in accordance with the Business Source License, use of
+// this software will be governed by the GNU Lesser General Public License v3.
+
+package file
+
+import (
+	"errors"
+	"fmt"
+	"github.com/Fantom-foundation/Carmen/go/backend/stock"
+	"github.com/Fantom-foundation/Carmen/go/backend/utils"
+	"go.uber.org/mock/gomock"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+func TestReadOnlyFile_Get(t *testing.T) {
+	const Items = 3300
+	directory := t.TempDir()
+	s, err := openInitFileStock(directory, Items)
```

Comment:
nit: wherever this function is defined, it should be renamed to `openAndInitFileStock` or `initFileStock` ; the way it is now looks like it is opening some special "InitFileStock" ...

Reply:
ok, I picked `initFileStock`